setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
+"statusCode", and "success". Some microservices may add an additional
+label such as "account" or "application" depending on the nature of
+the microservices API.
+
+These metrics will have several time series, such as those with the
+following tag bindings:
+
+account | controller | method | status | statusCode | success
+-----------|--------------|------------|----------|-----------------|-----
+my-aws-account | ClusterController | getForAccountAndNameAndType | 2xx | 200 | true
+my-aws-account | ClusterController | getForAccountAndNameAndType | 4xx | 404 | false
+my-aws-account | ClusterController | getForAccountAndNameAndType | 4xx | 400 | false
+                           | ApplicationsController | list | 2xx | 200 | true
+                           | ApplicationsController | get | 2xx | 200 | true
+                           | ApplicationsController | get | 4xx | 404 | false
+
+
+You can aggregate over the success tag to count successful calls vs failures,
+perhaps breaking out by controller and/or method to see where the failures
+were. You can break out by statusCode to see which controller and/or
+method the errors are coming from and so forth.
+
+Different metrics have different tags depending on their concept and
+semantics. Some of these tags may be of more interest than others. In
+the case above, some of the tags are at different levels of abstraction
+and not actually independent. For example a 2xx status will always be
+success=true and a non-2xx status code will always be success=false.
+Which to use is a matter of convienence but given the status tag (which
+can distinguish 4xx from 5xx errors) the success tag does not add any
+additional time-series permutations since its value is not actually
+independent.
+
+### About the Supplied Dashboards
+
+Each of the supplied monitoring solutions provides a set of dashboards
+tailored for that system. These are likely to evolve at different rates
+so are not completely analogous or consistent across systems and might
+not be completely consistent with the document. However, the gist and
+intent described here should still hold since the monitoring intent is
+the same across all the concrete systems.
+
+As a rule of thumb, the dashboards currently prefer showing value differences
+(over rates) for 1-minute sliding windows. This might change in the future.
+Some of the caveats here are due to the choice to show values over rates, but
+at this time the values seem more meaningful than rates, particularly where
+there arent continuous streams of activity. Where latencies are shown, they
+are computed using the counters from the past minute.
+
+Depending on the chart and underlying monitoring system, some charts show
+instantaneous value differences (between samples) while others show
+differences over a sliding window. Either way, the dashboards are not
+necessarily precise numbers but should reflect accurate trends and relative
```

Comment:
The phrase "not necessarily precise" when referring to metrics directly gathered from the system gave me pause. It makes the user beg the questions "Why aren't they precise? Is there a way to make them more precise? How do I get more precise numbers?"

Perhaps what you're meaning is that no metric system can be 100% accurate, but that the precision of the absolute values aren't as important as the relative differences over time.